.yarnrc.yml

```diff
@@ -0,0 +1,3 @@
+nodeLinker: node-modules
```

Comment:
how is upgrading storybook related to this change? 🤔

Reply:
This change took place when I upgraded yarn. A similar one exists on main repo: https://github.com/appsmithorg/appsmith/blob/release/app/client/.yarnrc.yml